2013-05-07  Aurelien Jarno <aurelien@aurel32.net>

	* misc/sys/param.h (DEV_BSIZE): Define only if not already defined.

diff --git a/misc/sys/param.h b/misc/sys/param.h
index 5e6353d..d257ec7 100644
--- a/misc/sys/param.h
+++ b/misc/sys/param.h
@@ -71,7 +71,9 @@
 
 
 /* Unit of `st_blocks'.  */
-#define DEV_BSIZE       512
+#ifndef DEV_BSIZE
+# define DEV_BSIZE	512
+#endif
 
 
 /* Bit map related macros.  */
-- 
1.7.10.4

